GUI/DeusEx/Classes/FrobDisplayWindow.uc

```diff
@@ -492,14 +497,90 @@ function string ComputersStrInfo(#var(prefix)ElectronicDevices d, out int numLin
     return strInfo;
 }
 
+function string WeaponStrInfo(#var(DeusExPrefix)Weapon w, out int numLines)
+{
+    local string strInfo;
+    local int dmg,inHandDmg,dmgDiff;
+    local string dmgDiffStr;
+    local float mod;
+    local #var(DeusExPrefix)Weapon ihw;
+
+    numLines=1;
+    strInfo = w.itemName;
+    if (w.AmmoName != Class'DeusEx.AmmoNone' )
+        strInfo = strInfo $ " (" $ w.PickupAmmoCount $ ")";
+
+    numLines++;
+    //Calculate the damage of the weapon you're looking at
+#ifdef injections
+    dmg = DXRWeapon(w).GetDamage(true);
+
+    if( class<DeusExProjectile>(w.ProjectileClass) != None && class<DeusExProjectile>(w.ProjectileClass).default.bExplodes==false )
+        dmg /= float(DXRWeapon(w).GetNumHits());
+#else
+    if (w.AreaOfEffect == AOE_Cone){
+        if (w.bInstantHit)
+        {
+            dmg = w.HitDamage * 5;
```

Comment:
why are we multiplying by number of hits here, but above in the #ifdef injections we divide by the number of hits?

Reply:
Differences in how the DXRWeapon GetDamage works compared to the vanilla one

Reply:
I still feel like it should be 1 difference but not both?

like a 5 damage shotgun, DXRWeapon will say 25 damage then you divide by 5 hits to display 5 damage

non-vanilla will return 5 HitDamage, but then you're multiplying by the 5 hits, so now you're displaying 25 damage?

Reply:
Yeah, this matches how the damage is shown in the inventory screen in non-vanilla (damage x num shots)

Reply:
Or at least in Revision

Reply:
oh ok yea, it's an intentional display difference